### PR TITLE
Updates to Read me

### PR DESCRIPTION
**Pull request by Kazleigh**
I have used the default settings from the pull request for the base and head repository so I hope this works - but should be updating the read me - Added imagery for terminal, command prompt and downloads section of git at 400px. Added short description of Git Bash, added 'following default settings' to git download description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,8 +72,9 @@ First, some quick vocab for y’all:
 - **Git:** is an open source VCS. Think of it like a program that runs on your computer that you can use in your coding projects to help manage versions of that project.
 	- Managing versions of code becomes especially important the more iterations the code goes through, the more complex/large the codebase becomes, and the more people you have working on it.
 	- Git is the most common VCS for code today, by a large margin.
-	- We will have a section in a bit digging in and understanding Git.
+	- We will have a section in a-bit digging in and understanding Git.
 	- Git is NOT GitHub.
+        - Git Bash is the name for the command-line terminal environment
```
Comment (wlfmnstr) [inline README.md:98]: This is a great point to add!

File: README.md [hunk 2]
```diff
@@ -167,4 +176,4 @@ On YOUR new forked repository (NOT MINE, use the one in your github account that
 
 The right way to do this is to first CLONE the repository (again, clone your fork, not my upstream original repo). Here are the steps to do this:
 
-1. Coming soon...
\ No newline at end of file
+1. Coming soon....
```
Comment (wlfmnstr) [inline README.md]: A very important additional `.`

File: student-resumes/index.html
Comment (wlfmnstr) [inline student-resumes/index.html:1]: I think you can delete this file.

**Review by wlfmnstr — CHANGES_REQUESTED** [review]
@Kazleigh Look good to me and thanks for the contribution! Only request is to delete the duplicate index.html at /student-resumes/index.html

File: student-resumes/index.html
Comment (Kazleigh) [reply student-resumes/index.html:1]: I have deleted it out and committed the change - so this should now be resolved

**Comment by Kazleigh** [comment]
The requested change has been complete as I deleted the extra file out and committed it - but it is still showing change requested and merging blocked and I am unsure how to resolve further or if requires your input? Sorry to be a pain! 

edit - I did also try closing the pull request and reopening to see if that would fix but no bueno.

**Review by wlfmnstr — APPROVED** [review]
lgtm! Thanks!

**Pull request merged**